common/datatest-stable/src/runner.rs

```diff
@@ -22,19 +22,91 @@ struct TestOpts {
     filter: Option<String>,
     #[structopt(long = "exact")]
     /// Exactly match filters rather than by substring
-    filter_exact: bool,
+    exact: bool,
     #[structopt(long, default_value = "32", env = "RUST_TEST_THREADS")]
     /// Number of threads used for running tests in parallel
     test_threads: NonZeroUsize,
     #[structopt(short = "q", long)]
     /// Output minimal information
     quiet: bool,
     #[structopt(long)]
-    /// We already can't capture anything but we don't want arg parsing to fail so this is a noop
+    /// NO-OP: don't capture stdout/stderr of each task, allow printing directly
```

Comment:
Its fine marking all of these options as noops but I think it will greatly distract from that fact that you've included and exact copy of the option's functionality.  Instead you should probably use the same wording for all unsupported options. Something like:
`NO-OP: unsupported option, exists for compatibility with the default test harness`

Reply:
Sure, don't mind do that.   I put extra effort in to providing the command's doc so that it'd be easier to add the functionality in the future in the run if it was desired.   I'll create a second commit in the pr for this with that exact wording for each no-op.